connectors/es/sink.py

```diff
@@ -802,26 +868,12 @@ async def cancel(self):
     def ingestion_stats(self):
         stats = {}
         if self._extractor is not None:
-            stats.update(
-                {
-                    "doc_created": self._extractor.total_docs_created,
-                    "attachment_extracted": self._extractor.total_downloads,
-                    "doc_updated": self._extractor.total_docs_updated,
-                    "doc_deleted": self._extractor.total_docs_deleted,
-                }
-            )
+            stats.update(self._extractor.counters.to_dict())
         if self._sink is not None:
-            stats.update(
-                {
-                    "bulk_operations": dict(self._sink.ops),
-                    "indexed_document_count": self._sink.indexed_document_count,
-                    # return indexed_document_volume in number of MiB
-                    "indexed_document_volume": round(
-                        self._sink.indexed_document_volume / (1024 * 1024)
-                    ),
-                    "deleted_document_count": self._sink.deleted_document_count,
-                }
-            )
+            stats.update(self._sink.counters.to_dict())
```

Comment:
So we extend the protocol to contain more stats? Might be worth to change README for the protocol in the repo too!

Reply:
Nope, the protocol has not changed here. The protocol just takes a subset of our counter keys, and that's defined here: 
https://github.com/elastic/connectors/blob/8b07cf3a268db9361062e0384aa67ddac4488647/connectors/protocol/connectors.py#L82-L87

I constantized this in my changes, but didn't actually change what we're storing. I think we eventually should persist more, but that's outside the scope of this current ticket.